### DISCUSSION TITLE
Thinking Cap #5

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/211A-ASIAAM-191A#16] [Assignments]
## Prompt:
> Do you think data justice and design justice relate? How can you share about these ideas with others beyond this class? 

### Note: Responses submitted after Tuesday 1pm will not be eligible for awards.

This week's thinking cap requires completing the [reading](https://github.com/albertkun/211A-ASIAAM-191A/blob/master/Week_05/Materials/DataJustice.pdf) beforehand.

## Submission
Make a new comment below in at least 3 - 4 sentences and respond to any two students posts before class begins.

**Comment by edmondywen** [comment]
Data, when improperly used, can magnify “social inequalities” like “maldistribution, misrecognition, and misrepresentation” because technology is situated within larger institutions of power (Dencik, Hintz, Reddin, Treré 2019). Therefore, I believe that data justice is a subset of design justice because the usage of data, whether that be in a project or report, ought to center the voices of those who are directly impacted and sustain communities so that standing power imbalances are challenged. One example of this mentioned in the reading is the usage of counter-mapping by indigenous peoples to counteract European settler narratives of ownership and control. My first step to sharing these ideas beyond the class is to first apply them within my own day to day work so that I am fully committed to data justice. By participating in the process of data justice, I am helping to normalize it as a practice and can also teach it to others when I am proficient enough to be a peer/mentor to others.

**Reply by cynthinee** [reply]
I also like the strategy of counter-mapping and counternarrative to disrupt mainstream, Eurocentric history and geography. I'm really bothered by the presentation of data and maps as neutral pieces of information that are actually heavily influenced by the conditions in which they were created. 

Something I'm working on that isn't so political in nature is thinking about how thermal comfort plays out in libraries. I think this relates to design justice! So library scholarship is all about what library researchers *think* libraries are about. We talk about how users find information, how they interact with librarians, user experience in regards to the library catalog and databases, etc. But there's really nothing about how library users use the library in terms of thermal experience. That's because librarians say they're about providing information access, but that kind of thinking makes us ignore the other ways in which users interact with these spaces. Are students coming to the libraries on campus to escape heat? Escape cold? How do students deal with the library being too cold? (Libraries are kept super cold for the preservation of books and other materials.) The only scholarship about heat in libraries is done by architects, engineers, HVAC specialists, etc., but they don't really have the experience of working with users after they build these systems. Could librarians maintain cold spaces for library books and other materials and less frigid spaces for humans to comfortable sit (room-by-room zoning)? Could libraries create Snuggie/blanket lending programs? (I hear this is a thing in at least one other library!) For my profession working in college libraries, I want to think more about the student experience and engaging students meaningfully in providing services and resources through dialogue and getting out of our assumptions about what libraries *ought* to be.

**Reply by eandrusenki** [reply]
I think we need to acknowledge, as you pointed out, that when data is used improperly, and even design work, can magnify social issues and that data and design justice must be a requirement/mandatory as to not undo hard work of social justice work. Therefore the idea of counter-mapping is extremely important and necessary to undo the work that has not been proven to be beneficial or important for Indigenous peoples.

**Reply by hrfrahn** [reply]
I never thought to frame data justice as a subset of design justice, but your analysis makes sense to me; the process of data collection is consciously designed by whoever's collecting the data, so it's just as subject to bias as any other form of design.

**Comment by cynthinee** [comment]
Data is always being collected for various reasons upon various communities. There's also the potential for data–what data isn't being collected, or not being collected in appropriate ways, and how can and should that data be collected, and by whom? Data can intentionally and unintentionally be used dangerously and irresponsibly and be oppressive. Data justice aims to undue these harms that have come from living in an increasingly data-driven world. Design justice similarly aims to dismantle structural inequalities and strive towards liberation. While data and design are intrinsically linked to communities, communities are often excluded, misrepresented, and ultimately harmed from both these fields who purport to help communities. Data justice and design justice are related in their attempts to actually help communities and strive towards actual solutions and liberation. 

With that said, I think both data justice and design justice can mean different things for different community members. I like the below quote for thinking about using data justice in this case as a framework against dominant data practices, questioning if liberation is even possible with how inherently messed up data practices have gotten, and as an aspirational thing that we strive towards. 

>  ...[W]e should use data justice as a form of critique, a framework for shifting the entry-point and debate on data-related developments in a way that foregrounds social justice concerns and ongoing historical struggles against inequality, oppression and domination. The question remains whether data infrastructures can ever be extracted and redirected from the current conditions of injustice. For others, however, holding on to a substantial notion of data justice as an ideal serves as a fruitful avenue to seek reforms that can better uphold justice claims pertaining to particular contexts. On this reading, data justice can advance principles and practices that emerge from existing conditions of possibility in order to facilitate processes that may lead to emancipatory outcomes. The aim of data justice in this sense, therefore, is to pinpoint where and how changes in data developments need to come about. 
> 
> (Dencik et al., p. 876)

For sharing with people outside of the course, I'd say is to really examine the relationship between data and design with communities and question if that relationship is a good one or not. I'd also emphasize looking at the conditions in which the data were created and to push back against the idea of data being neutral.

**Reply by ronnie-blanche** [reply]
Hi Cynthia, I really appreciate your point on how design and data justice looks very different for different community members! A lot fo the time when we discuss justice or combatting oppression,  conversation typically revolves around this uniformity of what justice could look like. Since oppression already differs between members of the same communities, there should also exist different actions to enact justice rather than a cookie cutter mold. The same  also goes to our understanding on design and date justice. Given that there are these different needs and problems within communities, do you think that these issues have some sort of correlation to the design/date justice needed to uplift their community.

**Reply by luigimagdamit** [reply]
I really like your point about data/design justice having different definitions between different people or groups of people. Even our beliefs about data or design justice are subject to subjectivity. I think its important to consider other ways of knowing in regards to data and design justice, because different cultures could have different practices in regards to these.

**Comment by abi-200** [comment]
This week’s reading helped me expand my own definition of what exactly data justice means, and it made me think more critically about not only data ethics and use, but also about the wide variety of disciplines intertwined with data and about how different areas of thoughts have diverse and complex associations with what it means to practice data justice. Last week’s reading on design justice helped me contextualize just how prevalent design is in our everyday life, and this week’s reading felt like an extension such that it discusses how the data that will then be used in design must be worked with or critiqued under a framework. In this way, I believe that data justice and design justice are related such that data justice is a first (and then ongoing) steop, and design justice should be used to enquire equity of data use and to not uphold structural problems that can use data as justification for it. 
In particular, I found it interesting how the paper in some ways separated the terms “data” and the word” justice.” For example, the paper mentions grassroots organizations primarily in the context of justice, and connected data justice to ethical data use such that the use of data should ensure that collected information is not used to further marginalize vulnerable communities (ex. Used to justify extra policing, etc). In this way, I believe that this is also connected to design justice such that data on low-income status or POC status of certain communities can contribute to social and political designs that keep these communities under scrutiny and discriminiation. The other discussed issue of the EDGI dataset was also very interesting to me because in this class, and in the paper, one of the tenets of data justice is its use to empower marginalized communities and make data more open-sourced and citizen-oriented.. However, this example showcases how there may be ethical implications with certain scientific data being open to more people.
One more way data justice connects to design justice as mentioned in the paper has to do with the idea of technological determinism; that is, the paper mentions how it is ironic how people often cite technological innovations as potential solutions to social issues despite the fact that technology has often served to increase division and discriminiation in any case. In this way, data and design justice are linked because while technology should certainly be discussed in both contexts, it must be taken into account alongside the bigger picture to make sure that it is more so a tool used for community empowerment rather than to increase inequalities. Lastly, the Nadia example shows how inclusion in data interface implementation does not translate to justice in practiced design (digital divide).
To share with people beyond this class, I believe that since many of our future paths will lead us to collaborate in projects some way or another (in professions, graduate school etc), we can critically ask ourselves and our peers what our design and collected data actually do to uphold our own definitions of justice and to help the communities in question.

**Reply by ramirezirvin** [reply]
I agree, this week's reading helped me have a more in-depth understanding of these topics and see how they could relate to one another. This sure broadens my view on how to view my future design and data collection for future projects to come. Yet I knew the differences of income status, ethnic identity but reminded me also about other intersectionalities that may lay within that, such as disabilities that may exclude other individuals in project and data representation. This can really help one think and be mindful of how to be all-inclusive in their work.

**Comment by ronnie-blanche** [comment]
I would say that both data justice and design justice are related and interconnected as they "broaden the terms of the debate in a way that accounts for a host of issues . . . the entrenchment and introduction of inequalities, discrimination and exclusion of certain groups, deteriorating working conditions . . . and interaction around sensitive issues" (Dencik et al., 874). An ongoing issue that we've been discussing with GIS and mapping is how our current understanding of what maps/mapping and data gathering is very centered around an academic and very Western/Eurocentric notion of what's important and what GIS/data should look like. An example that I really appreciate is when our first guest speaker spoke about how Native culture is often seen as inferior or not legitimate in terms of their mapping practices. Another point I really enjoyed is Cynthia's posts posts on how a lot of academic circles disregard the contribution of the actual communities, community-based solidarity, and academic contributions made my people who don't hold high positions in academic circles. However, data justice and design justice work together to recenter the importance of marginalized communities, forgotten voices, and bringing light to the inequalities that exist within data collecting that stems from our society. I personally have been incorporating data and design justice in my own life by taking a critical stance on the information I've been learning in my classes, trying to call out power imbalances and injustices when I see them, but also working towards being a more socially aware person overall.

**Reply by abi-200** [reply]
I see this as a really important point - the need to dewesternize our view of datafication is definitely necessary if we hope to ever study certain systems while prioritizing the lens of those actually affected. For example, the paper talks about the India's Aadhar biometric system that was said to be used to collect data to identify vulnerable groups to provide them with aid - it is likely that in the eyes of those who grew up in the United States or a western country with similar data policies, this system is highly unlikely to pass a vote. While corruption is a valid concern with regards to the biometric data collected on every citizen, the perception of Aadhar in India is not that of widespread outrage as may be expected elsewhere. The data here is not neutral, and is subject to the same data justice principles we may consider, but it is important to take into account others' perceptions of data justice when creating a baseline framework of what is and is not a "good" use of data.

**Reply by edmondywen** [reply]
I really like how you integrate previous class topics into your post. I definitely agree with how academic circles can oftentimes leave community contributions on the back burner, thus reinforcing societal inequalities. I feel like academics have an incentive to produce work for citations and reputation within academia which unfortunately reinforces the ivory tower nature of the university.

**Comment by ramirezirvin** [comment]
It is noted that within data collection, specifically early data collection the primary people in this data collection were cis-heterosexual white males usually higher-middle class, making the majority of the data that was collected skewed. Thus having to have data be recollected in further years for other genders and ethnic groups, as the data collected before would be inaccurate to modern-day society. As design justice goes with the same thing, the primary people in charge of design were white individuals because other people of color had to earn their place in society it was not given. Even when creating new projects they design things mainly geared towards what they call the "majority" in society being people who are caucasian, because the main people working on these projects are white people and usually since they don't know how to provide a space for other poc they are disregarded in the design process because of eurocentricism, this while also excluding people who may have disabilities, their income status and other intersectionalities that may lie within that. The majority of things that are maid are done to benefit them only. There is always a lack of awareness and representation of people of color, and in data, there is a lack of people of color being targeted for the data to be collected. For collecting demographic data there was a point where each individual ethnic group was not recognized in the census and slowly became recognized due to constant activism to fight to be seen and heard, and people who are of lower class often being excluded in this data collection. This is why both data and design justice were created to help broaden their view, to help out other individuals be included in projects, and to better the data and design to be all-inclusive and easily accessible to all who may be interested to take part in said project or data collection.

**Reply by abi-200** [reply]
I agree that the complete lack of diversity in data collection is a massive problem given we use this data in guiding decisions even today (ex. healthcare research regarding the different between male and female heart attack symptoms, etc). However, I like how you brought up the point about the same being true with regard to design. Before last week's reading, I did not give too much thought into how design structures (beyond broader political/social structures) are made to overrepresent and underrepresent some groups. In terms of literal design, there are a few notable examples that are more widely known, such as the death risk disparity between genders with seat belt testing, but it is abysmally common just how little the overall diversity of test subjects is taken into account in many studies currently conducted.

**Reply by ronnie-blanche** [reply]
Hi Irvin, I really appreciate how you pointed out the centering of white, cis, and straight experiences within data collection and forming, and the fact that different communities have historically been left out. When we think of data, especially sets that are meant to be transformative and how data is collected, we often think of them as equally representing all targeted communities. So it's really great that you pointed out the reality of data being very white and male centered, as well as how marginalized communities need more representation and quality when it comes to this.

**Comment by eandrusenki** [comment]
Design justice and data justice are definitely related, and I think they even intertwine in some aspects. With both, there are parameters you need to take into consideration to make sure that you understand how your work will positively change social justice issues but also understand the implications and effects it will have on people, and the community specifically. This reading, like others, along with the class, encourages diversity in data collection, and "decentering" technology. This is because a lot of technology has western bias based on the people who are in power and control technology. It is important that the people in power are not majority shite straight males. Even in the U.S. there is diversity, therefore even in the Western world, Western views can't apply. Only a small amount of people can benefit if there is not design or data justice. One way to encourage design justice and data justice is to be willing to step back in projects where I am not a member of the community and be willing to share my own knowledge. Another way is to hold other people accountable, and speak up when I see injustices in data collection and design.

**Reply by hrfrahn** [reply]
I totally agree that both design and data justice are necessary to build truly democratic projects (like we're trying to do in this class!). Also, your point about stepping back and letting community members take the lead is a very important one, and something that I think a lot of people (me included) coming from a more academic context can struggle with.

**Reply by emilysuh** [reply]
I really like that you mentioned accountability! I think community-based accountability models can teach us a lot about how to approach, understand, and solve interpersonal and social problems. Your comment made me think about how democratic (community-based) approaches to accountability and to data are interlinked!

**Reply by edmondywen** [reply]
Love the point about diversity in data collection. I believe that this class's emphasis on using community-driven sources of data is vital to decentering institutional power and norms which can overlook marginalized voices. If this is not done, data science will be caught in a feedback loop where biased data collection informs biased data analysis which furthers bias throughout decision-making and therefore society at large.

**Reply by ramirezirvin** [reply]
I totally agree with you on the importance of not having the people in power always be straight white males, and to have more diverse people involved. As you mentioned in the western world we are diverse, now we just have to find a way to break these norms in society and give a chance to diversify the people in charge and make that the new norm in society. Certainly society needs to normalize these changes to better these communities and create more exposure over these injustices and not be apologetic about deserving what is needed.

**Comment by hrfrahn** [comment]
It seems pretty clear that data justice and design justice are closely related, and many aspects of/approaches to one of the two also apply to the other. For example, the Dencik reading mentions how data
> is not primarily a technical question but relates to long-standing social, political, economic and cultural issues.

This refusal to allow data to be "de-politicized" and treated as "objective" (when in reality it's just as subject to bias as anything else created by people) immediately reminded me of the reading we did about design justice, which emphasized the importance of acknowledging how things are _consciously_ designed a certain way (and for a certain purpose/audience) rather than just created devoid of any context whatsoever. In the same way, data and design both increasingly dominate our lives: almost everything we deal with on a daily basis is shaped by conscious design, and "big data" powers the technologies that we have come to rely on. 

Data & design justice are powerful tools for understanding how our modern world is shaped. I think the best way to share these approaches with people outside of the classroom is to use them as methods of critique and to remind us that no data, no matter how scientific/objective it may seem, is immune to bias.

**Reply by emilysuh** [reply]
Hi Harry! I appreciate how you discuss objectivity and bias as it relates to design and data. Science itself is up for debate on whether it can really reveal the truth about our world -- there are ongoing philosophical discussions on whether the scientific method/ways of reasoning are actually valid. I think statistics is a really interesting example in thinking about this (how do we make larger inferences from data?). Of course, not saying that science is useless/false (I'm vaccinated and masked!), but I agree with you we need to think very critically about what we know to be "objective."

**Reply by perezsara59** [reply]
I agree and especially with data being seen as something that can be "objective" I think its important to state the outcomes even more clearly so the data is not misinterpreted. To some people who may not understand the way data is being presented because of some possible barrier(language), its even more important to incorporate ideas of design justice and ensure that barrier is not the reason they dont fully understand the data and as a result misinterpret it.

**Reply by eandrusenki** [reply]
I am glad you brought up and reminded us that instead of trying to get rid of biases within technology, which will always be present, we should fully acknowledge the biases and then we can try to figure out how technology should or should not be used. It may be too optimistic to try to remove human bias from technology, and understanding the bias in conjuncture with empowering community members and passing on knowledge may ne the best way to encourage design and data justice.

**Reply by luigimagdamit** [reply]
I think your point on treating data as something less objective is interesting. I like to think that data itself IS design. Perhaps we dictate the design of the way data is shaped, thus making it completely subjective to our attempts at making an objective depiction of the world.

**Comment by emilysuh** [comment]
I think data justice and design justice are closely related, and both frameworks center structural inequalities and the stratification of power in their analysis of how design or data work and impress upon the world. I appreciated the notion of "cognitive injustice" in the data justice reading, which was used to clarify how "data is seen as an avenue to revert or challenge dominant understandings of the world, (re)creating conditions of possibility for counter-imaginaries and social justice claims to emerge." I believe the same can be said for design, which prefigure and pre-organize our knowledge and actions. 

Part of sharing these justice-oriented frameworks of data and design with the world outside of this class is to be thoughtful about how I engage with data and knowledge production as a student at a major USian research institution. What does it mean, for example, when I am asked to write ethnographies on communities I'm not a part of, within a 2-3 week period, in order to "demonstrate my understanding of course materials"? One solution is to honestly discuss and name my discomfort and subjective standing as a student researcher who is not from this community within that assignment.

**Reply by cynthinee** [reply]
I really resonate with your last part about being asked to assert some level of authority when doing research on communities of which we're not a part. I think that stating your discomfort and your standing is something that is becoming more common in academic research spaces. In my field, positionality statements are becoming more normative, which is great. I think these kinds of statements are great ways that we can be upfront about our limitations and assumptions and show epistemic humility.

**Reply by perezsara59** [reply]
Hi Emily, I agree that the notion of cognitive justice can be very powerful. I wonder if the data that is now being challenged lacked community input. Now, you have community input countering what the community itself knew all along but was ignored. I also think you pointing out your experience and considering being more honest when writing ethnographies and stating your discomforts is a good conversation to have and include in ethnographies especially when its with a community you may not be comfortable with.

**Comment by perezsara59** [comment]
I think data justice and design justice relate in that they both encourage more equitable, ethical, and inclusive maps. Data justice more specifically, focuses on how data is used, collected, and generated. This is important because if researchers or whoever is gathering data does so in a way that is invasive or exploitive, its not empowering to the community. Not just community based but community led research shifts the power to the community and focuses on what they deem important rather than what an outsider thinks they need or want to extract from them. Design justice strives for accessibility and inclusivity which in conjunction with data justice, result in more ethical approaches. Beyond this class, I can incorporate less invasive and more community-based or community-led approaches to data collection, use, and analysis.

**Comment by luigimagdamit** [comment]
I think its really interesting to consider the direct relationship of design justice and data justice. As a stats major and someone that will probably end up doing some form of data analytics(if I get my degree lolol), data visualization can have a massive effect on our perceptions of the existing data. I think this concept is similar to Mariah's presentation on maps. Some representations of data through design can be very misleading and sometimes downright oppressive  I think that being able to understand the relationship between these two can really boost our awareness of the current world and help us to take steps to undermine our modern maladies.
